src/common.jl

```diff
@@ -97,7 +97,7 @@ for func in (:(==), :isequal, :isapprox)
 
         for f in fields
             isdefined(s1, f) && isdefined(s2, f) || return false
-            $func(getfield(s1, f), getfield(s2, f); kwargs...) || return false
+            getfield(s1, f) === getfield(s2, f) || $func(getfield(s1, f), getfield(s2, f); kwargs...) || return false
```

Comment:
Why is this change needed?

Reply:
This is convenient if we have parameters that can be `missing` by design. Since `missing == missing` produces `missing`, the default equality checks, `==`, `isapprox`, etc defined between distributions fail for `missing` parameters. But since `missing === missing` (and IIRC equivalence is always defined), then with this change those checks still pass.

Reply:
I see, can you add a comment?

Reply:
Sure!